light-zk.js/src/transaction/sendVersionedTransaction.ts

```diff
@@ -68,12 +68,14 @@ export const sendVersionedTransaction = async (
     tx = await payer.signTransaction(tx);
     try {
       return await connection.sendTransaction(tx, confirmConfig);
-    } catch (e: any) {
-      console.log(e);
+    } catch (error: any) {
+      console.log(error);
+
       retries--;
-      if (retries == 0 || e.logs !== undefined) {
-        console.log(e);
-        throw e;
+      if (retries == 0 || error.logs !== undefined) {
+        console.log(error);
+
+        throw error;
```

Comment:
I believe this is the part that we threw the cryptic errors, and it looks unchanged to me, is this on purpose?